docs/30DaysOfPWA/advanced-capabilities/06.md

```diff
@@ -1,15 +1,174 @@
-# 2.6: Synchronizing app data in the background
+# 2.6: Synchronizing app in the background
 
-## *Coming soon!*
+![Placeholder Banner Only. Replace when final assets ready.](_media/day-01.png)
 
-![Placeholder Banner Only. Replace when final assets ready.](_media/week2-placeholder.jpg)
+Welcome to day 14 of the [30 Days of PWA](https://aka.ms/learn-pwa/30Days-blog) series! Today's blog post will introduce Background Services of modern browsers and how to use them for different kinds of synchronizations.
 
-### WHAT WE'LL COVER TODAY
+Author: Maxim Salnikov [@webmaxru](https://twitter.com/webmaxru)
 
-| Section | Description |
-| ------- | ----------- |
-| **Definition** | Can I sync data without my PWA being open? |
-| **Differentiation** | What are my options for syncing data? |
-| **Adoption** | What are some good examples of sites syncing data in the background? |
-| **Exercise** | Create a sync task. |
-| **Related** | Look for other advanced capabilities throughout [Week 2: Advanced Capabilities](../advanced-capabilities). |
+## Background services
+
+Modern browsers and Service Worker API bring exciting new features to the web. Now, you can run some pieces of your frontend code completely independently from the main application lifecycle. That means you are no longer limited to architecting and building a frontend application as something that always starts when the users open the corresponding url and immediately ends right after they close the tab. Now, parts of your code can be executed when there is no tab with your application open and even when the browser (to be more precise - its visible part) is not open, i.e. in the background. With or without notifying the user - it's you decide as a developer.
```

Comment:
```suggestion
Modern browsers and Service Worker APIs bring exciting new features to the web. Now, you can run some pieces of your frontend code completely independently from the main application lifecycle. That means you are no longer limited to architecting and building your application as something that only starts when the user opens the corresponding URL and immediately ends right after they close the tab. Now, parts of your code can be executed even when there is no browser window with your application open in it, i.e. in the background. And this can happen with or without notifying the user. It's you, as a developer, who decide.
```

Reply:
Just a suggestion to make this a little bit simpler for our users.